API.yaml

```diff
@@ -1430,6 +1510,50 @@ components:
           description: The bookmark's unique ID, generated by the server.
           type: integer
           format: int128
+    Annotation:
+      type: object
+      properties:
+        start:
+          description: the start time for this annotation.
+          type: integer
+          format: int64
+        end:
+          description: the end time for this annotation. (Optional)
+          type: integer
+          format: int64
+        label:
+          description: this annotation's name (Optional)
+          type: string
+        category:
+          description: The type of the annotation (generic, output, ...)
+          type: string
+        iconUrl:
+          description: URL to the annotation's icon (Optional)
+          type: string
+        UUID:
+          description: The annotation's unique ID, generated by the server.
+          type: integer
+          format: int128
+    AnnotationCategory:
```

Comment:
AnnotationCategory needs a recommended field or something like that to tell if this category should be shown by default